Documentation/SubmittingPatches

```diff
@@ -135,8 +135,10 @@ The body should provide a meaningful commit message, which:
 [[imperative-mood]]
 Describe your changes in imperative mood, e.g. "make xyzzy do frotz", 
 as if you are giving orders to the codebase to change its behavior.
-Do not describe you changes in present or past tense, e.g. "[This patch] makes xyzzy do frotz" or "[I] changed xyzzy
-to do frotz". Try to make sure your explanation can be understood
+Do not describe you changes in present or past tense, 
```

Comment:
you*r* changes, right?

Please do wrap the commit message at 72 or 76 columns per line. Also, the second commit is essentially a fixup for the first one. Please use [`git rebase -i` to squash the second into the first](https://git-scm.com/docs/git-rebase#_interactive_mode): no need to send a known-incorrect patch to the Git mailing list, together with a patch that immediately fixes it.

Reply:
Oh, and I also forgot to mention that the commit message should start with a prefix, in this case I think `SubmittingPatches: ` would be excellent.